GVFS/GVFS.Platform.Windows/WindowsFileSystemVirtualizer.cs

```diff
@@ -1034,15 +1034,14 @@ private HResult QueryFileNameHandler(string virtualPath)
                                 HResult writeResult = this.virtualizationInstance.WriteFile(streamGuid, targetBuffer, (ulong)writeOffset, bytesToCopy);
                                 remainingData -= bytesToCopy;
 
-                                if (writeResult != HResult.Ok)
+                                if (writeResult != HResult.Ok &&
+
+                                    // StatusFileClosed is expected, and occurs when an application closes a file handle before OnGetFileStream

```

Comment:
Minor, can you update this comment to say `FileClosed` instead of `StatusFileClosed` (the old text is referring to an enum that we no longer have in our code).

Reply:
Done!